recipes/kipoi/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "kipoi" %}
+{% set version = "0.2.8" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "8cde375de1ce3930645ea8915dceea5617fc741d3934af2e584e6878876cff19" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  entry_points:
+    - kipoi = kipoi.__main__:main
+  script: pip install 'related>=0.6.0' enum34 epc && python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - git-lfs
+#    - libstdcxx-ng # [not osx]
+    - gcc # [not osx]
+    - llvm # [osx]
+    - python
+    - pip
+    - setuptools
+    - pyyaml
+    - future
+    - numpy
+    - pandas
+    - tqdm
+#    - related >=0.6.0
+#    - enum34
+    - colorlog
+    - cookiecutter
+    - h5py
+  run:
+#    - libstdcxx-ng # [not osx]
+    - libgcc  # [not osx]
+    - git-lfs
+    - python
+    - pyyaml
+    - future
+    - numpy
+    - pandas
+    - tqdm
+#    - related >=0.6.0
+#    - enum34
+    - colorlog
+    - cookiecutter
+    - h5py
+
+test:
+  imports:
+    - kipoi
+    - kipoi.cli
+    - kipoi.external
+    - kipoi.external.related
+    - kipoi.external.torch
+    - kipoi.postprocessing
+    - kipoi.postprocessing.variant_effects
+    - kipoi.postprocessing.variant_effects.utils
+  commands:
+    - "kipoi --help"
+  requires:
```

Comment:
Better?

Reply:
yup. perfect (sorry for the late response)

Reply:
no problem. I was also busy with other things...

Reply:
Could you update the version to 0.3.6?

Two more questions:
- how do we increase the version on bioconda: submit another PR?
- kipoi has an additional set of dependencies (https://github.com/kipoi/kipoi/blob/master/setup.py#L58-L67) not installed by default. Is there a way to setup an alternative/extended set of dependencies on conda like there is in the setup.py?

Reply:
sure. already noticed that 0.3.3 was not the latest version.

About your question:
- version: Yes. New PR. If you change the recipe and not the version you have to increase the build number
- additional dependencies: I have no idea how we should implement that. I will ask Johannes or Björn about that. In theory we can add a `build.sh` script which contains a short command like `kipoi-install-extended-libraries` (or similar). It will use conda to install the additional libraries. Also a kipoi and a kipoi-extended bioconda recipe might be possible...

Reply:
Thanks! Let me know if you think of any good solution. I could also try splitting the kipoi package into two: kipoi, kipoi-vep in the spirit of say pytest where the other packages containing - are additional modules.